frontend/validator.js

```diff
@@ -34,14 +35,18 @@ var Validator = (function() {
     req.send(JSON.stringify({ schema: content, }));
   }
 
+  /**
+   * Fetch from remote SpaceAPI endpoint
+   */
   function validate_url(ev) {
     banner.className = 'fetching';
     ev.preventDefault();
     offlineCheck();
 
     var url = getValue(url_input);
     if (url.indexOf('http://') === -1 && url.indexOf('https://') === -1) {
-      appendLog('\nInvalid URL: '+url);
+      appendLog('Invalid URL: '+url);
+      setBanner('ERROR', 'Invalid URL: '+url);
```

Comment:
As far as I know, there is currently no other transport method allowed. (See https://github.com/spacedirectory/meta/issues/5)
There used to be `gopher://` and `ftp://`.
`file://` is not allowed for security reasons.